src/imageLoader/wadouri/metaData/metaDataProvider.js

```diff
@@ -121,6 +121,44 @@ function metaDataProvider (type, imageId) {
       }
     };
   }
+
+  if (type === 'overlays') {
```

Comment:
To be consistent w/ the other modules in this `metaDataLoader`, this should probably be `overlayPlaneModule`

Reply:
done, also I comitted the change to createImage I forgot !